flink-dist/src/main/resources/META-INF/NOTICE

```diff
@@ -0,0 +1,62 @@
+flink-dist
+Copyright 2014-2018 The Apache Software Foundation
+
+This product includes software developed at
+The Apache Software Foundation (http://www.apache.org/).
+
+This project bundles the following dependencies under the Apache Software License 2.0 (http://www.apache.org/licenses/LICENSE-2.0.txt)
+
+- com.twitter:chill-java:0.7.6
+- com.twitter:chill_2.11:0.7.6
+- com.typesafe:config:1.3.0
+- com.typesafe:ssl-config-core_2.11:0.2.1
+- com.typesafe.akka:akka-actor_2.11:2.4.20
+- com.typesafe.akka:akka-camel_2.11:2.4.20
+- com.typesafe.akka:akka-protobuf_2.11:2.4.20
+- com.typesafe.akka:akka-slf4j_2.11:2.4.20
+- com.typesafe.akka:akka-stream_2.11:2.4.20
+- commons-cli:commons-cli:1.3.1
+- commons-collections:commons-collections:3.2.2
+- commons-io:commons-io:2.4
+- org.apache.camel:camel-core:2.17.7
+- org.apache.commons:commons-compress:1.4.1
+- org.apache.commons:commons-lang3:3.3.2
+- org.apache.commons:commons-math3:3.5
+- org.javassist:javassist:3.19.0-GA
+- org.objenesis:objenesis:2.1
+- org.rocksdb:rocksdbjni:5.7.5
+- org.xerial.snappy:snappy-java:1.1.4
+
+This project bundles the following dependencies under the BSD license.
```

Comment:
I would combine the BSD licenses in one section (BSD-style licenses). The BSD license names are too varied that I would feel comfortable spelling the specific variant out.

Reply:
Sounds good. Will change it.